includes/functions/llms-functions-content.php

```diff
@@ -112,6 +112,24 @@ function llms_get_post_sales_page_content( $post, $default = '' ) {
 
 }
 
+/**
+ * Retrieve the decimal value for average column in Reporting.
+ *
+ * By default the database return floating-point values and  function
+ * percision to round off the decimal value.
+ *
+ * @since [version]
+ *
+ * @return int
+ */
+function llms_get_reporting_round_precision() {
```

Comment:
Also, I don't feel this is the right file for this helper.
In this file we have helpers related to page/post's "post_content" alteration.

This function should be moved somewhere else.

Reply:
done!